app/code/core/Mage/Adminhtml/Block/Widget/Grid/Column/Renderer/Theme.php

```diff
@@ -66,7 +66,7 @@ public function getOptions()
         }
         if (empty($options) || !is_array($options)) {
             $options = Mage::getModel('core/design_source_design')
-                ->setIsFullLabel(true)->getAllOptions(false);
+                ->setIsFulllabel(true)->getAllOptions(false);
```

Comment:
will this still continue to work? 
if this property is magic, then changing the case will result in different property being set.

Reply:
@tmotyl is right. Seems to break functionality ...

app/code/core/Mage/Core/Model/Design/Source/Design.php

    /**
     * Setter
     * Add package name to label
     *
     * @param bool $isFullLabel
     * @return Mage_Core_Model_Design_Source_Design
     */
    public function setIsFulllabel($isFullLabel)
    {
        $this->_isFullLabel = $isFullLabel;
        return $this;
    }

Reply:
Following the naming convention of getter and setter.

I think is better rename the function/method from setIsFulllabel to setIsFullLabel

Reply:
@sreichel How do you think this breaks anything? It's simple correcting the case of the function name being called to the proper case. PHP doesn't enforce this so it currently works being mis-cased.

Reply:
After investigating this a little bit, this is NOT breaking.

Reply:
@LeeSaferite Wasn't aware the just variables are case sensitive, and methods are not ...